install/kubernetes/cilium/templates/cilium-operator/clusterrole.yaml

```diff
@@ -82,9 +82,11 @@ rules:
   - get
   - list
   - watch
+{{- if or .Values.ingressController.enabled (and .Values.bgpControlPlane.enabled .Values.bgp.announce.loadbalancerIP) }} 
+  - update
```

Comment:
Jus to be sure, do we need to update endpoints as well? Why?

Reply:
No, that was an oversight. I discovered that we have a rule which grants 'update' permissions to `services/status` for the MetalLB. Turns out it didn't work since I patch the status. This change also wouldn't have worked after the code switched from update to patch. 

I have updated the LB rule to also include patching.

Reply:
I'd suggest just always allowing patch + update, and drop the Helm conditional. It simplifies enabling this in day-2 scenarios.

(In fact, I'm arguing for removing the enable flag all together, but that's a separate thread :-) )